libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/MapboxDistanceFormatter.kt

```diff
@@ -35,7 +35,8 @@ class MapboxDistanceFormatter private constructor(
     private val context: Context,
```

Comment:
This is `private`, why isn't it a `data class`?

Reply:
the typical pattern in many of the builders is to have a getter function. But this is a good point, the data class objects should be

data class Example(
   val foo: String,
   val bar: Int
) {
 // getter for foo is redundant. the constructor declaration is a read only `val`

}

Reply:
I was referring to `MapboxDistanceFormatter` 😬 👀 https://github.com/mapbox/mapbox-navigation-android/pull/2624#discussion_r403145619 cc @cafesilencio